src/services/codefixes/fixUnusedIdentifier.ts

```diff
@@ -100,7 +98,7 @@ namespace ts.codefix {
                     // or "'import {a, b as ns} from './file'"

```

Comment:
Can you change this to use the `|` delimiters like the other comments in this file?